plugins/helphub-contributors/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "bugs": {
+    "url": "https://github.com/zzap/HelpHub-Contributors/issues"
+  },
+  "description": "WordPress plugin for tracking contributors to wordpress.org Documenation team's HelpHub project.",
+  "devDependencies": {
+    "autoprefixer": "^7.1.2",
+    "grunt": "~1.0.1",
+    "grunt-check-dependencies": "~1.0.0",
+    "grunt-checktextdomain": "~1.0.1",
+    "grunt-contrib-copy": "^1.0.0",
+    "grunt-contrib-jshint": "~1.1.0",
+    "grunt-contrib-watch": "^1.0.0",
+    "grunt-jscs": "~3.0.1",
+    "grunt-jsvalidate": "~0.2.2",
+    "grunt-postcss": "^0.8.0",
+    "grunt-sass": "^2.0.0",
+    "grunt-stylelint": "~0.9.0",
+    "matchdep": "~2.0.0",
+    "select2": "^4.0.3",
+    "stylelint": "~8.0.0",
+    "stylelint-config-wordpress": "~12.0.0",
+    "stylelint-scss": "^2.0.1"
+  },
+  "engines": {
+    "node": ">=6.9.1"
+  },
+  "keywords": [
+    "community",
+    "helphub",
+    "support",
+    "wordpress"
+  ],
+  "homepage": "https://github.com/zzap/HelpHub-Contributors",
+  "license": "GPL-2.0+",
+  "name": "HelpHub-Contributors",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/zzap/HelpHub-Contributors"
+  },
+  "version": "1.0.0"
+}
```

Comment:
There shouldn't be a need to add this file, any required changes or additions should be added to the main project `package.json` located at https://github.com/Kenshino/HelpHub/blob/master/package.json